File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        let numSet = new Set(nums)
        let long = 0;
        for(let num of numSet) {
            if(!numSet.has(num-1)) {
                let length = 1;
                while(numSet.has(num+length)) {
                    length++;
                }
                long = Math.max(long,length)
            }
        }
        return long
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-2.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        map_ = dict()

        for i in range(len(nums)):
            key = target-nums[i]
            if key in map_:
                return [map_.get(key),i]
            map_[nums[i]] = i
        return [-1,-1]
